hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/Gateway.java

```diff
@@ -51,6 +53,11 @@ public Void call() throws Exception {
     TracingUtil.initTracing("S3gateway", ozoneConfiguration);
     OzoneConfigurationHolder.setConfiguration(ozoneConfiguration);
     ozoneConfiguration.set("hadoop.http.authentication.type", "simple");
```

Comment:
I don't think we need to hard-code "hadoop.http.authentication.type"="simple", but this is a separate issue that we can fix in a different JIRA.

Reply:
That's correct, I will open another JIRA to remove this hard-code configuration.